server/github/builder.go

```diff
@@ -12,20 +14,20 @@ import (
 // ImageBuilder is an interface that represents something that can build and
 // return a Docker image from a GitHub commit.
 type ImageBuilder interface {
-	BuildImage(ctx context.Context, event events.Deployment) (image.Image, error)
+	BuildImage(ctx context.Context, w io.Writer, event events.Deployment) (image.Image, error)
```

Comment:
I'm actually not sure what the standard way to do this in Go is, but would it be useful to have some documentation on what these arguments are?  Like it's not immediately obvious what `w` is used for here.